src/frequenz/sdk/actor/power_distributing/_component_managers/_battery_manager.py

```diff
@@ -127,8 +127,10 @@ class BatteryManager(ComponentManager):
     def __init__(
         self,
         component_pool_status_sender: Sender[ComponentPoolStatus],
+        results_sender: Sender[Result],
```

Comment:
No parameter descriptions at all? :o
Why does our CI even allow this :)

Reply:
Ah, because it's in the abstract class?

Reply:
No,  I think @llucax made parameter descriptions optional, so that we can have property documentation or something related.  So if no parameter documentation is provided at all, CI doesn't complain anymore, but still catches issues if it is partial or outdated, etc.

I will add the descriptions.